musl only provides off64_t, not __off64_t.
glibc and uclibc have both defined.

Signed-off-by: Alex Suykov <alex.suykov@gmail.com>

--- a/F2CLIBS/libf2c/sysdep1.h
+++ b/F2CLIBS/libf2c/sysdep1.h
@@ -10,7 +10,7 @@
 
 #ifdef __linux__
 #define USE_LARGEFILE
-#define OFF_T __off64_t
+#define OFF_T off64_t
 #endif
 
 #ifdef _AIX43
